node-operators/appchain-node/rpc-systemd.md

```diff
@@ -0,0 +1,265 @@
+
+- **Appchain specifications file** - the Appchain specification file is needed to run the node. You can download it from the dashboard in the [dApp](https://apps.tanssi.network/){target=\_blank} by clicking the `Appchain Data` link
+
+    ![Getting the chain specs](/images/node-operators/appchain-node/rpc-systemd/rpc-systemd-1.webp)
+
+- **Relay chain specifications file** - the relay chain specification file can be downloaded from this [public GitHub repository](https://github.com/papermoonio/external-files/blob/main/Moonbeam/Moonbase-Alpha/){target=\_blank}
+
+## Download the Latest Release {: #download-latest-release }
+
+Every new release includes two different node binaries, one for EVM-compatible Appchains and another one for Substrate Appchains. To get started, run the following command to get the latest release binary that matches your Appchain type and make it executable:
+
+=== "EVM-Compatible Appchain"
+
+    ```bash
+    wget https://github.com/moondance-labs/tanssi/releases/latest/download/container-chain-template-frontier-node && \
+    chmod +x ./container-chain-template-frontier-node
+    ```
+
+=== "Substrate Appchain"
+
+    ```bash
+    wget https://github.com/moondance-labs/tanssi/releases/latest/download/container-chain-template-simple-node && \
+    chmod +x ./container-chain-template-simple-node
+    ```
+
+!!! note
+    Optimized binary versions for [Skylake](https://www.intel.com/content/www/us/en/products/platforms/details/skylake-u-y.html){target=\_blank} and [Zen3](https://www.amd.com/en/technologies/zen-core){target=\_blank} architectures are also available in the [releases](https://github.com/moondance-labs/tanssi/releases/latest){target=\_blank} page.
+
+## Download the Relay Chain Specs File {: #download-relay-specs }
+
+The node binary file includes also the necessary code to run a relay chain node. When launching your Appchain's node, it will also be required to provide the relay chain's specification file as a parameter. 
+
+Download the relay chain specification file executing:
+
+```bash
+wget https://raw.githubusercontent.com/papermoonio/external-files/main/Moonbeam/Moonbase-Alpha/westend-alphanet-raw-specs.json
+```
+
+## Setup the Systemd Service {: #setup-systemd-service }
+
+[Systemd](https://systemd.io/){target=\_blank} is a management system for Linux systems that manages services (daemons in Unix-like systems jargon), starting them automatically when the computer starts or reboots, or restarting them upon unexpected failures.
+
+It is a good practice to have the service running with its own non-root account and grant that account writing access to a specific directory. Run the following commands to configure a new account and the directory:
```

Comment:
I think this should be stated earlier tbh

Reply:
[Moved to the intro]